pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -2159,6 +2159,7 @@ func parseConfigItems(config *zconfig.EdgeDevConfig, ctx *getconfigContext) {
 	// That way bringup of new hardware models can be done using an
```

Comment:
Makes sense to update this comment to also mention VgaAccess on L2154 and screen on L2160